src/patterns/data/default/exposure_to_environmental_material.tsv

```diff
@@ -104,12 +104,12 @@ ECTO:7000117	exposure to fine respirable suspended particulate matter	ENVO:01000
 ECTO:7000118	exposure to ultrafine respirable suspended particulate matter	ENVO:01000416
 ECTO:7000119	exposure to contaminated water	ENVO:00002186
 ECTO:7000120	exposure to pulp-bleaching waste water	ENVO:00002193
-ECTO:7000121	exposure to fibrous glass dust	ENVO:02000102
```

Comment:
why was this removed?

Reply:
There were duplicate terms for 'exposure to fibrous glass dust' and it still remains as ECTO:7000017